.github/workflows/default.yml

```diff
@@ -21,7 +21,10 @@ jobs:
             - run: yarn install
               env:
                   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-            - run: yarn lint
+            # There is a weird bug in typescript eslint where if fails if you are linting an eslint
+            # plugin with a config that uses that same eslint plugin. We need the latest typescript eslint versions
```

Comment:
Not sure what was the intention of the first sentence 👯, but I guess that we can't use ESlint to lint this plugin

Reply:
TLDR: A uses B, B uses A. Circular dep that I guess eslint plugin doesn't like. I should probably open an issue about that at some point.